Content.Client/Cabinet/ItemCabinetVisualizer.cs

```diff
@@ -15,12 +16,15 @@ public class ItemCabinetVisualizer : AppearanceVisualizer
         [DataField("emptyState", required: true)]
         private string _emptyState = default!;
 
+        [DataField("state", required: true)]
+        private string _baseState = default!;
```

Comment:
base state doesn't need to be specified if presumably it never gets changed, it can just be part of the spritecomp and the visualizer changes the stuff above it